homeassistant/components/sonos/media_player.py

```diff
@@ -1075,6 +1078,11 @@ def set_option(self, data):
         if ATTR_SPEECH_ENHANCE in data and self._speech_enhance is not None:
             self.soco.dialog_mode = data[ATTR_SPEECH_ENHANCE]
 
+    @soco_error()
+    def play_queue(self, data):
+        """Start playing the queue."""
+        self.soco.play_from_queue(0)
```

Comment:
Should we make that "0" configurable, i.e. extend the schema with an optional "position" attribute?

Reply:
Excellent idea, will add.